openquakeplatform/test/shape_test.py

```diff
@@ -31,10 +30,18 @@ def upload_shape_test(self):
         pla.wait_new_page(uploadlayer, '/layers/upload', timeout=10)
 
         # choose shape file
+        sph = os.path.join(os.path.expanduser("~"), 'oq-platform2',
+                                                    'openquakeplatform',
+                                                    'test',
+                                                    'shapefile',
+                                                    'exampleshape.zip')
+
+        browser = webdriver.Firefox()
+
```

Comment:
Does is browser used in some place ?

Reply:
Fixed here: https://github.com/gem/oq-platform2/commit/db1bbf03619757dcaaba97d991552aec090a132d